bcda/auth/client/ssas.go

```diff
@@ -118,8 +118,29 @@ func (c *SSASClient) GetPublicKey(systemID int) ([]byte, error) {
 }
 
 // ResetCredentials PUTs to the SSAS /system/{systemID}/credentials endpoint to reset the system's credentials.
-func (c *SSASClient) ResetCredentials(systemID int) ([]byte, error) {
-	return nil, nil
+func (c *SSASClient) ResetCredentials(systemID string) ([]byte, error) {
+	req, err := http.NewRequest("PUT", fmt.Sprintf("%s/system/%s/credentials", c.baseURL, systemID), nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to delete credentials")
+	}
+
+	resp, err := c.Do(req)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to delete credentials")
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 201 {
```

Comment:
```suggestion
	if resp.StatusCode != http.StatusCreated {
```
To be consistent with the rest of our code, it might make sense to use the HTTP status code constant here.